README.md

```diff
@@ -224,6 +229,13 @@ if (true !== violation) {
   console.log('"foo" is not a valid IP. Violation:', violation[0].show());
   // => "foo" is not a valid IP. Violation: { assert: 'Ip', value: 'foo' }
 }
+
+// Make the validation nullable.
+violation = validator.validate(null, is.nullOr(is.ip()));
```

Comment:
I added it because it's slightly different than the others, since it can be applied to any other assert. So it can be applied to the example assert. If it's not that useful, I'm ok with removing the example.